src/main/java/seedu/address/logic/filters/combinator/FilterCombinator.java

```diff
@@ -22,19 +22,17 @@
  * Customer}.
  */
 public class FilterCombinator implements Predicate<Customer> {
-    public static final String MESSAGE_EFFECTIVELY_EMPTY = "Your string contains only space of brackets, which is "
-        + "invalid.";
+    public static final String MESSAGE_EFFECTIVELY_EMPTY = "Your string contains only space of brackets "
```

Comment:
I think this is better:
```suggestion
    public static final String MESSAGE_EFFECTIVELY_EMPTY = "Your string contains only spaces or brackets "
```